ansible/plugins/inventory/nodejs_yaml.py

```diff
@@ -45,10 +45,10 @@
     'type': ('infra', 'release', 'test'),
 
     # providers - validated for consistency
-    'provider': ('azure', 'digitalocean', 'joyent', 'ibm', 'linuxonecc',
+    'provider': ('azure', 'digitalocean', 'joyent', 'linuxonecc',
                  'macstadium', 'marist', 'mininodes', 'msft', 'osuosl',
                  'rackspace', 'requireio', 'scaleway', 'softlayer', 'voxer',
-                 'packetnet', 'nearform')
+                 'packetnet', 'ibm', 'nearform')
```

Comment:
Any particular reason why `ibm` was moved here? It was already in the set so this change is not needed, or if it is changed maybe this should be sorted alphabetically?

Reply:
Good catch my fault, mustve moved it when I changed it to pcc and forgot to move it back